spec.md

```diff
@@ -300,6 +300,14 @@ attributes and the event data will be materialized. For example, in the case
 of a JSON serialization, the context attributes and the event data might
 both appear within the same JSON object.
 
+CloudEvent producers MAY include additional extension attributes within the
+event. Each specification that defines how to serialize a CloudEvent will
+define how extension attributes will appear. When defining a new extension
```

Comment:
Need to clarify what is meant by "include additional extension attributes within the event"? Do you mean Event producer can include any random attributes (and give it a name the producer likes) within the top level event attributes? If so, how could the event consumers know the semantics of those attributes? How could the event consumers know how to decode the random attributes (are they strings or ints ?) and properly use them?

Reply:
your question applies to all extensions no matter where they appear.